### ISSUE TITLE
Ender 2 Stock Display stays blank

### ISSUE THREAD

**Issue by Rushmonk** [bigtreetech/BIGTREETECH-SKR-E3-DIP-V1.0#8]
Dear All,

I'm using the SKR E3 DIP on my Ender 2, working all fine except for the display. It stays blank and I can't change it to a mini display. Could you please get some support for this?

**Comment by kleinerbiber** [comment]
In theory it could be used with the following pin configuration when you define it as MINIPANEL instead of CR10_STOCKDISPLAY, but I can't get it to work either. (I don't know if the bottom part is needed because I can't check it.)
Maybe somebody from @bigtreetech could look into this with my information given?

```
/**
 *          MINIPANEL (Ender-2)
 *                 _____
 *     (SCK) PA15 | · · | PB6 (BTN_ENC)
 * (BTN_EN1)  PA9 | · · | RST
 * (BTN_EN2) PA10 | · · | PB9 (LCD_CLK)
 *  (LCD_CS)  PB8 | · · | PB7 (MOSI)
 *            GND | · · | 5V
 *                 -----
 *                 EXP1
 */

  #define DOGLCD_SCK     PA15
  #define DOGLCD_MOSI    PB7
  #define DOGLCD_CS      PB8
  #define DOGLCD_A0      PB9

  #define BTN_ENC        PB6
  #define BTN_EN1        PA9
  #define BTN_EN2        PA10

  // Alter timing for graphical display
  /**
   * #ifndef ST7920_DELAY_1
   *   #define ST7920_DELAY_1 DELAY_NS(125)
   * #endif
   * #ifndef ST7920_DELAY_2
   *   #define ST7920_DELAY_2 DELAY_NS(125)
   * #endif
   * #ifndef ST7920_DELAY_3
   *   #define ST7920_DELAY_3 DELAY_NS(125)
   * #endif
   */
```

**Comment by tobybonhof** [comment]
I want this aswell! Any updates?

**Comment by kleinerbiber** [comment]
> 
> 
> I want this aswell! Any updates?

Support from BTT for their products = -1

**Comment by tobybonhof** [comment]
Yeah doing a huge file dump on github and calling it opensource.... yeah right

**Comment by choco107** [comment]
Any updates on this, need it also

**Comment by kleinerbiber** [comment]
Ender-2 Display seems to be supported now (defined as `MKS_MINI_12864`)! - Too bad I already sold my board again.
[https://github.com/MarlinFirmware/Marlin/pull/15924/](url)

**Comment by choco107** [comment]
Nothing shows on that link ??

**Comment by kleinerbiber** [comment]
> Nothing shows on that link ??
Sorry, fixed it. URL wasn't linked properly.

**Comment by choco107** [comment]
Aw great , does merged mean it's in the latest bugfix download ??

**Comment by kleinerbiber** [comment]
> Aw great , does merged mean it's in the latest bugfix download ??

Yes!

**Comment by choco107** [comment]
Fantastic I'll try it tomorrow

**Comment by oldman4U** [comment]
Hi.

What is the status of this issue? 

@Rushmonk Could you please close this ticket in case you do not need it anymore.

Thank you

**Comment by oldman4U** [comment]
@bigtreetech Could you please close this unneeded ticket. Thank you

**Comment by oldman4U** [comment]
Please help the community and close the ticket once you do not need it anymore. Thank you

**Comment by oldman4U** [comment]
Please help the community and close the ticket once you do not need it anymore. Thank you

**Comment by oldman4U** [comment]
Please help the community and close the ticket once you do not need it anymore. Thank you

**Comment by oldman4U** [comment]
Please help the community and close the ticket once you do not need it anymore. Thank you

**Comment by kleinerbiber** [comment]
> @oldman4U 
> Please help the community and close the ticket once you do not need it anymore. Thank you

Please stop annoying everybody by spamming this message, thanks.

**Comment by oldman4U** [comment]
Please help me to make this ticket system useful by getting rid of unneeded tickets and comments like yours.

Thank you

PS. You can delete all your comments in this ticket to get no notifications anymore.

**Comment by oldman4U** [comment]
Please help the community and close the ticket once you do not need it anymore. Thank you

**Comment by oldman4U** [comment]
Please help the community and close the ticket once you do not need it anymore. Thank you
